.circleci/config.yml

```diff
@@ -144,7 +144,7 @@ jobs:
                     -kubecontext="kind-dc1" \
                     -secondary-kubecontext="kind-dc2" \
                     -debug-directory="$TEST_RESULTS/debug" \
-                    -consul-k8s-image=docker.mirror.hashicorp.services/hashicorpdev/consul-k8s:latest
+                    -consul-k8s-image=gcr.io/nitya-293720/consul-k8s-dev:metrics30
```

Comment:
reminder to remove

Reply:
Could we merge it with this into feature-metrics so acceptance tests on the branch can stay green, and then when we merge feature-metrics into master we could remove it?

Reply:
oh yeah of course!